test/tests_util.c

```diff
@@ -0,0 +1,9 @@
+#include "tests_util.h"
+#include <stdlib.h>
+
+#ifndef HAVE_RANDOM
+int random(void)

```

Comment:
Some platforms (Windows) don't have the random() function. Here we create our own.